core/src/com/unciv/CrashScreen.kt

```diff
@@ -0,0 +1,113 @@
+            
+            
+            """.trimIndent()
+    }
+
+    init {
+        println(text) // Also print to system terminal.
+        stage.addActor(Table().also { layoutTable ->
+        layoutTable.width = stage.width
+            layoutTable.height = stage.height
+            layoutTable.add(
+                "An unrecoverable error has occurred in Unciv:".toLabel(fontSize = 24).also {
+                    it.wrap = true
+                    it.setAlignment(Align.center)
+                }
+            ).padBottom(15f)
+                .width(stage.width)
+                .row()
+            layoutTable.add(
+                AutoScrollPane(Table().also {
+                    it.add(Label(text, skin).apply {
+                        setFontSize(15)
+                    }).pad(10f)
+                        .row()
+                }).addBorder(4f, Color.DARK_GRAY)
+            ).maxWidth(stage.width * 0.7f)
+                .maxHeight(stage.height * 0.5f)
+                .minHeight(stage.height * 0.2f)
+                .row()
+            layoutTable.add(
+                "{If this keeps happening, you can try disabling mods.}\n{You can also report this on the issue tracker.}".toLabel().also {
+                    it.wrap = true
+                    it.setAlignment(Align.center)
+                }
+            ).padTop(15f)
+                .width(stage.width)
+                .row()
+            layoutTable.add(Table().also { buttonsTable ->
+                buttonsTable.add(
+                    "Copy".toButton()
+                        .onClick {
+                            Gdx.app.clipboard.contents = text
+                            copied = true
+                            ToastPopup(
+                                "Error report copied.",
+                                this@CrashScreen
+                            )
+                        }
+                ).pad(10f)
+                buttonsTable.add(
+                    "Open Issue Tracker".toButton(icon = "OtherIcons/Link")
+                        .onClick {
+                            if (copied) {
+                                Gdx.net.openURI("https://github.com/yairm210/Unciv/issues")
+                            } else {
+                                ToastPopup(
+                                    "Please copy the error report first.",
+                                    this@CrashScreen
+                                )
+                            }
+                        }
+                ).pad(10f).also {
+                    if (isCrampedPortrait()) {
+                        it.row()
+                        buttonsTable.add()
+                    }
+                }
+                buttonsTable.add(
+                    "Close Unciv".toButton()
+                        .onClick {
+                            Gdx.app.exit()
+                        }
+                ).pad(10f)
+            }).padTop(10f)
```

Comment:
This is kind of horrendous to read. 
Please use `val`s to separate the parts, call them by name, activate functions etc